README.md

```diff
@@ -52,6 +52,8 @@ See more design systems [here](https://github.com/alexpate/awesome-design-system
 - [Design system checklist](https://designsystemchecklist.com/) - Build better design systems. An open-source checklist to help you plan, build and grow your design system.
 - [Design.system](https://design.systems/) - Resources for the design systems community.
 - [Super friendly](https://superfriendlydesign.systems/) - We help in-house teams make better digital products with design systems.
+- [Design Systems Survey](https://designsystemssurvey.seesparkbox.com/) - Designers and developers from more than 20 industries tell us about their design systems and their uses.
+
```

Comment:
🙌 good!! 

Can we sort it alphabetically? Thanks for the contribution!

Reply:
no problem - have sorted alphabetically ✅

Reply:
Thanks!!! 👏